Common.mk

```diff
@@ -410,17 +438,21 @@ validate-checksums: $(BINARY_TARGETS)
 	$(BUILDCTL)
 
 # Build helm chart
+ifeq ($(HAS_HELM_CHART),true)
 .PHONY: helm/build
-
+helm/build: ## Build helm chart
 helm/build: $(OUTPUT_DIR)/ATTRIBUTION.txt
```

Comment:
Does this work separating the pre-reqs like this?  if so, yeah it is def cleaner.